### PR TITLE
Updating set-up page to include link to Ignition set-up documentation… and provided a brief description paragraph to go with it.

### PR DESCRIPTION
**Pull request by sabor413**
… in GIT repository

### CODE REVIEW

File: docs/Setup.rst [hunk 1]
```diff
@@ -1,3 +1,11 @@
 *****
 Setup
 *****
+
+The Ignition framework is a .NET MVC based architecture development framework used to create a coded infrastructure to support a Sitecore based website.   Sitecore is a content management website development framework.  Ignition provides a rapid development mechanism for creating components used to render pages using Sitecore as the website framework.
```
Comment (coreyasmith) [inline docs/Setup.rst]: I don't think this is necessary in the _Setup_ section as it's already in the index of the docs.

File: docs/Setup.rst [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+Information regarding setting up the Ignition framework is provided through the README file associated with the Sitecore Ignition GIT repository accessable through `this link`_.
```
Comment (coreyasmith) [inline docs/Setup.rst]: GIT -> GitHub
accessable -> accessible

File: docs/Setup.rst [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+
+.. _this link: https://github.com/sitecoreignition/SitecoreIgnition/
+
+Please continue to use the above link for any update to the installation process concerning the ignition framework.
```
Comment (coreyasmith) [inline docs/Setup.rst]: ignition -> Ignition

**Comment by coreyasmith** [comment]
I made comments directly on the Files changed. Push those changes and I'll approve the pull request.

**Comment by coreyasmith** [comment]
Push fixes to the comments above and I'll approve the pull request.

**Comment by sabor413** [comment]
I made the changes you requested Corey, feel free to review again...

File: docs/Setup.rst [hunk 4]
```diff
@@ -2,8 +2,6 @@
 Setup
 *****
 
-The Ignition framework is a .NET MVC based architecture development framework used to create a coded infrastructure to support a Sitecore based website.   Sitecore is a content management website development framework.  Ignition provides a rapid development mechanism for creating components used to render pages using Sitecore as the website framework.
-
 Information regarding setting up the Ignition framework is provided through the README file associated with the Sitecore Ignition GITHub repository accessible through `this link`_.
```
Comment (coreyasmith) [inline docs/Setup.rst]: GITHub -> GitHub

**Comment by sabor413** [comment]
Made change, ready for review...

**Pull request merged**